exonum/src/blockchain/tests.rs

```diff
@@ -653,19 +655,19 @@ fn test_dispatcher_register_rollback() {
         None
     );
     // Tests that an unavailable artifact will not be registered.
-    let artifact_id: ArtifactId = ServicePanicImpl.artifact_id().into();
-    execute_transaction(
-        &mut blockchain,
-        TestDeploy { value: 24 }.sign(TEST_SERVICE_ID, keypair.0, &keypair.1),
-    );
-    let snapshot = blockchain.snapshot();
-    assert!(!dispatcher::Schema::new(snapshot.as_ref())
-        .artifacts()
-        .contains(&artifact_id.name));
-    assert_eq!(
-        Entry::<_, u64>::new(IDX_NAME, snapshot.as_ref()).get(),
-        None
-    );
+    let error_string = panic::catch_unwind(panic::AssertUnwindSafe(|| {
```

Comment:
Maybe this statement be broken into a few lines with the help of intermediate variable for better readability?